### PR TITLE
Implemented basic Bootstrap in all active views

### PR DESCRIPTION
**Pull request by bandreasen**
Pretty much that. Let me know what else you want me to do!

### CODE REVIEW

File: client/ViewPage.js [hunk 1]
```diff
@@ -34,8 +34,8 @@ Vue.component('view-page', {
 		<div id="view-page">
 			<div id="view-links">
 				<p><a href="#" v-on:click="$emit('page', 'search')">Back to search results</a></p>
-				<p><button v-on:click="editSnippet">Edit this snippet</button></p>
-				<p><button v-on:click="deleteSnippet">Delete this snippet</button></p>
+				<p><button class="btn" v-on:click="editSnippet">Edit this snippet</button></p>
+				<p><button class="btn" v-on:click="deleteSnippet">Delete this snippet</button></p>
```
Comment (allout58) [inline client/ViewPage.js]: I would recommend a `.btn-danger` as well

File: client/index.html [hunk 2]
```diff
@@ -5,6 +5,15 @@
 	<meta charset="UTF-8">
 	<title>CheatSheet</title>
 
+	<!-- Latest compiled and minified CSS -->
```
Comment (allout58) [inline client/index.html]: v3 not v4?

**Comment by timschwab** [comment]
Jaaaaaaaaaaames Hollowell!

Yeah, I'd say just switch to Bootstrap 4 for this PR.

Next PR can be these things:

- make the delete button `.btn-danger`
- add margins to the sides and top of the page
- on the add/edit pages, reduce the size so that the horizontal scrollbar doesn't appear

But it looks like a great beginning!

**Comment by timschwab** [comment]
And favorite part: the <code> formatting looks just perfect. Bootstrap FTW

**Comment by allout58** [comment]
You might not just be able to "switch" to BS 4 from 3, some of the class names changed if I remember correctly. I can take a look at a PR for that later if you want

**Comment by timschwab** [comment]
Yep, I was unclear. By “switch”, I mean do all work necessary to move from 3 to 4. If you want to, that’d be awesome! And there is a Trello team for CheatSheet that was a few boards, one of them being pre-alpha to v1.0. All we have left for 1.0 is Bootstrap stuff. So if you’d want to have a crack at it, I would love that!

File: client/AddForm.js [hunk 3]
```diff
@@ -38,14 +38,30 @@ Vue.component('add-form', {
...
-			<span>Problem:</span> <input type="text" v-model="problem">
-			<br />
-			<span>Solution:</span> <textarea rows=10 cols=40 v-model="solution"></textarea>
-			<br />
-			<span>Keywords:</span> <input type="text" v-model="keywordString"> {{ keywords }}
-			<br />
-			<input type="button" value="Submit" v-on:click="submit">
+		<div class="add-form form-horizontal">
+			<div class="form-group">
+				<label for="problem" class="control-label col-sm-2">Problem:</label>
```
Comment (allout58) [inline client/AddForm.js]: iirc BS 4 doesn't need any classes on the label

File: client/AddForm.js [hunk 4]
```diff
@@ -38,14 +38,30 @@ Vue.component('add-form', {
...
+			</div>
+			<div class="form-group">
+				<label for="keywords" class="control-label col-sm-2">Keywords:</label>
+				<div class="col-sm-10">
+					<input class="form-control" id="keywords" type="text" v-model="keywordString"> {{ keywords }}
+				</div>
+			</div>
+			<div class="form-group">
+				<div class="col-sm-offset-2 col-sm-10">
+					<input class="btn" type="button" value="Submit" v-on:click="submit">
```
Comment (allout58) [inline client/AddForm.js]: I personally prefer `<button class="btn">Submitt</button>` to this

Comment (timschwab) [reply client/AddForm.js]: Agreed

File: client/AddForm.js (hunk 3 above)
Comment (timschwab) [reply client/AddForm.js]: I am leaving this to figure out in v1.0

**Pull request merged**